modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -0,0 +1,145 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.apache.commons.io.IOUtils;
+import org.testcontainers.containers.delegate.CassandraDatabaseDelegate;
+import org.testcontainers.delegate.DatabaseDelegate;
+import org.testcontainers.jdbc.ext.ScriptUtils;
+import org.testcontainers.jdbc.ext.ScriptUtils.ScriptLoadException;
+import org.testcontainers.utility.MountableFile;
+
+import javax.script.ScriptException;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+/**
+ * Cassandra container
+ * <p>
+ * Supports 2.x and 3.x Cassandra versions
+ *
+ * @author Eugeny Karpov
+ */
+public class CassandraContainer<SELF extends CassandraContainer<SELF>> extends GenericContainer<SELF> {
+
+    public static final String IMAGE = "cassandra";
+    public static final Integer CQL_PORT = 9042;
+    private static final String CONTAINER_CONFIG_LOCATION = "/etc/cassandra";
+    private static final String USERNAME = "cassandra";
+    private static final String PASSWORD = "cassandra";
+
+    private String configLocation;
+    private String initScriptPath;
+
+    public CassandraContainer() {
+        super(IMAGE + ":latest");
+    }
+
+    public CassandraContainer(String dockerImageName) {
+        super(dockerImageName);
+    }
+
+    @Override
+    protected void configure() {
+        optionallyMapResourceParameterAsVolume(CONTAINER_CONFIG_LOCATION, configLocation);
+        addExposedPort(CQL_PORT);
+        setStartupAttempts(3);
```

Comment:
please move it to the constructor, so that it will be overridable

Reply:
Done. Thanks! 
Also many containers have this line of code and it just overwrites any value that could be set through withStartupAttempts method. Think it's undesirable behavior.